### ISSUE TITLE
Rich content can be hidden

### ISSUE THREAD

**Issue by gregoriopellegrino** [w3c/publ-a11y#671]
In Rich content section I see:

> This group should be displayed only if the metadata indicates the presence of math, chemical formulas, extended descriptions, videos, or transcripts of audio within the title, otherwise it can be hidden.

In fact, this group may be hidden if there is no metadata related to "enriched" content for accessible mathematical formulas, accessible video, or accessible audio.

I would rewrite it as:

> This group should be displayed only if the metadata indicates the presence of accessible math, accessible chemical formulas, extended descriptions, captions or transcriptions for videos, or transcripts of audio within the title, otherwise it can be hidden.
